courses/cunix/ex04/src/my_itoa.c

```diff
@@ -0,0 +1,43 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include "test.h"
+
+int get_digit_count (int nmb)
+{
+    int count = -1;
+    for (;nmb != 0; count++, nmb /= 10)
+    {}
+    return count;
+}
+
+char *my_itoa (int nmb)
```

Comment:
same for this huge func, try to avoid it in the future with for ex. splitting your code to smaller func